data/hai/hai reveal 5 philosophers.txt

```diff
@@ -1101,23 +1146,26 @@ mission "Hai Reveal [B13] Freedom"
 				goto finished
 			label knowhai
 			`	They incline their head respectfully. "The Hai are perhaps the greatest valuers of truth among us. If they have set you to this, then they will be honored by your diligence.`
+				goto finished
 			label knowsyndi
 			`	They focus on you closely. "The Syndicate is perhaps the most impressive machine ever wrought by man. It is worthy of both respect and fear, and will do to people as any machine does to the product of its design. Treat it as such and it will always do precisely what you expect it to; both good and bad.`
+				goto finished
 			label knowduty
 			`	They lift their head and smile. "A promise made is a promise kept. If your word is your bond then it would not do for me to stand in its way.`
+				goto finished
 			label finished
 			`	"The missing leader you seek is embedded within the Syndicate and is known by the name Terry Adrianopoulos."`
 			choice
-				`	(Be surprised.)`
+				`	"Terry did this!?"`
```

Comment:
```suggestion
				`	"Terry did this?"`
```

Reply:
The player is supposed to be surprised. Removing the exclamation point eliminates the feeling of surprise.

Reply:
If you don't like the interrobang, then the only option is to split up the sentence: "Aghast! Terry did this?"

Reply:
Splitting up the sentence would reduce the feeling of surprise, and an interrobang is the correct option here. If you can come up with another way to express surprise and ask a question in few words, then I'm game for it.

Reply:
I'm not really passionate about the change, I'm fine with it as is.